tldr.sh

```diff
@@ -207,21 +223,32 @@ exit 0
 while [ $# -gt 0 ]
 do
     case "$1" in
+        -h|-\?|--help)
+            usage
+            ;;
+        -d|--download)
+            pindex=$(cat $configdir/index.json)
+            parse
+            ;;
         -l|--list)
-            >&2 echo -e "Known tldr pages: \n"
-            tr '{' '\n' < "$configdir/index.json" | cut -d '"' -f4 | column
```

Comment:
even after 20 years of unix `column` is a command that I'm still only half aware of.